vignettes/testing_demo.Rmd

```diff
@@ -184,36 +184,29 @@ c("FLAG_increase_tests_7day", "FLAG_negative_tests_7day") %>%
-    not reported but OWID does obtain daily new tests) and a 7-day average 
-    of new daily tests is calculated from this. The short-term test positivity rate is in 
-    most cases based on the ratio of 7-day average of new daily cases (from JHU) 
-    divided by the 7-day average of new daily tests. For a handful of countries. 
-    OWID is able to get the test positivity rate directly from the same source as 
-    number of tests, and we use this pre-computed value for the positivity rate in 
-    these cases as it may be more reliable. Note they changed their 
-    algorithm in Jan 2022 to take any number of days between observations for interpolation, see 
-    [here](https://github.com/owid/covid-19-data/blob/master/scripts/scripts/testing/smoother.R). 
     
-    + For FIND we manually interpolate the cumulative number of tests, new daily tests
-    and the 7-day average of new daily tests to match the OWID outputs.
-    FIND scrapes websites and testing data sources once daily for a given date, 
+    + We interpolate the cumulative number of tests, and calculate the new daily tests
+    and the 7-day average of new daily tests based on this interpolated metric.
+    
+    + The maximum number of days we allow between two 
+    cumulative testing values for interpolation is 31 days(note that this can be 
+    changed by the param `find_maxgap` in `get_testing_long). Test positivity is 
+    based on the ratio of 7-day average of new daily cases (from JHU) to 7-day 
+    average of new daily tests. 
+    
+    + FIND scrapes websites and testing data sources once daily for a given date, 
     thus if the original source does not update, the scraped value is the same. 
     To be precise, if a the scraped cumulative test number has not changed on 
     subsequent days, we interpret this as no update to the number of tests (that 
     is, the original value from FIND provided for "new tests" is 0). To interpolate
-    in a manner consistent with OWID, we first only consider an update to have
+    in a consistent manner across the two datasets, we first only consider an update to have
     occurred if the cumulative testing number has actually changed. So if the 
     values are recorded as 90, 100, 100, 100, 100, 200 for Sunday through Friday,
     respectively, the interpolated series would be 90, 100, 125, 150, 175, 200. 
-    The maximum number of days we allow between two 
-    cumulative testing values is 31 days(note that this can be changed by the param 
-    `find_maxgap` in `get_testing_long). Test positivity is based on the ratio 
-    of 7-day average of new daily cases (from JHU) to 7-day average of new daily 
-    tests. 
+
+    + For a handful of countries,OWID is able to get the test positivity rate 
+    directly from the same source as number of tests, and we use this pre-computed 
+    value for the positivity rate in these cases as it may be more reliable. 
```

Comment:
I kept this in because I think from looking at the code it might be true? But do correct me if I'm wrong on this one too Sean!

Reply:
Yes, I think that's accurate! Thanks